### PR TITLE
add cypress tests

### PR DESCRIPTION
**Pull request by garzillo**

### CODE REVIEW

File: cypress/integration/search_spec.js [hunk 1]
```diff
@@ -1,25 +1,109 @@
...
+
+        it('has a menu button and export button', function(){
+            cy.search(good_term);
+            cy.get("#menuButton").click();
+            cy.get(".applauncher-pf-link").click({ multiple: true, force: true });
+        })
+
+        it('results have some information item element [abstract,sentence,table]', function () {
+            cy.search(good_term);
+            cy.get(".article-title").click({ multiple: true, force: true })
```
Comment (freshottr) [inline cypress/integration/search_spec.js]: for functions that you invoke multiple times, like "clicking article title" you should consider adding a custom `cypress` function as was done for `search(term)` for example

File: cypress/integration/home_spec.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    })
+
+    it('contains have an empty input box', function() {
+        cy.get('input').should('not.have.value', 'US')
+    })
+
+    // TODO: figure out how the date and count will be implemented and check for those things
+    it('displays a last updated date and article count', function(){
+        cy.get('#whatsnew').should('contain', 'Last update')
+        cy.get('#update-count').should('contain','new articles added') //any number
```
Comment (freshottr) [inline cypress/integration/home_spec.js]: We should probably display a fixed number of articles in the "What's new section". I'd think the most recent 10 or 20 would be good. From my point of view, the what's new mostly serves to give the user an idea of the currency of the data in our database.

File: cypress/integration/home_spec.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    })
+
+    it('whats new table has a header', function(){
+        cy.get('#table1_wrapper').should('contain', 'PMC ID')
+    })
+
+    // TODO: should we have an export button here?
```
Comment (freshottr) [inline cypress/integration/home_spec.js]: I wouldn't think so. Export should only be applicable after a search has been performed

File: cypress/integration/search_spec.js [hunk 4]
```diff
@@ -1,25 +1,109 @@
...
+        it('include authors for every row', function () {
+            cy.search(good_term);
+            cy.get(".article-title").click({ multiple: true, force: true})
+            cy.get('.pm-list-row-authors')
+            .each(function (el) {
+                console.log(el.context)
+            })
+        })
+
+        it('include a close button for every row', function () {
```
[the remark this answers is not in the document]
Comment (garzillo) [reply cypress/integration/search_spec.js]: Yeah, all the Cypress functions have implicit assertions (ie., exists, is clickable, etc).

**Pull request merged**